shared/naturalcrit/codeEditor/codeEditor.jsx

```diff
@@ -431,7 +433,7 @@ const CodeEditor = createClass({
 				}
 				text = foldPreviewText || `Lines ${from.line+1}-${to.line+1}`;
 
-				text = text.trim();
+				text = text.trim().replace('{', '').trim();
```

Comment:
Is this functionally different from `text = text.replace('{','').trim();`?

Reply:
In this context? Probably not. 

Didn't seem to be in my tests so I consolidated.